schulcloud/DashboardViewController.swift

```diff
@@ -133,42 +154,79 @@ final class DashboardViewController : UICollectionViewController {
     }
 
     override func numberOfSections(in collectionView: UICollectionView) -> Int {
-        return 1
+        return Globals.currentUser!.permissions.contains(.newsView) ? 2 : 1
     }
 
     override func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        return self.currentDesign == .extended ? viewControllers.count : viewControllers.filter({ (vc) -> Bool in
-            guard let vc = vc as? DashboardNoPermissionViewController else { return true }
-            return vc.missingPermission != .notificationView
-        }).count
+        guard let sectionType = Sections(rawValue: section) else { return 0 }
+
+        switch sectionType {
+        case .dashboard:
+            return self.currentDesign == .extended ? viewControllers.count : viewControllers.filter({ (vc) -> Bool in
+                guard let vc = vc as? DashboardNoPermissionViewController else { return true }
+                return vc.missingPermission != .notificationView
+            }).count
+        case .newsList:
+            return self.newsArticleFetchedController.fetchedObjects?.count ?? 0
+        }
     }
 
     override func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        let vc = viewControllers[indexPath.row]
-        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "DashboardCollectionCell", for: indexPath) as! DashboardCollectionViewControllerCell
-        cell.configure(for: vc)
-        vc.didMove(toParentViewController: self)
+        let sectionType = Sections(rawValue: indexPath.section)!
+        let cell : UICollectionViewCell
+
+        switch sectionType {
```

Comment:
There should only be one prototype cell for the dashboard. So the dashboard vc does not have to care about the actual displayed content.